tools/spellcheck/correct_words.txt

```diff
@@ -763,6 +784,12 @@ ltnote
 ###################################
 # common technical words {{{
 ###################################
+FHWM
+datasets
+Fortran
+NETLIB
+nonrobust
+rubstifying
```

Comment:
what kind of word is that?!
In the context its used, maybe it could be left out or replaced by something else
```The number of rubstifying iterations```